homeassistant/components/swisscom/device_tracker.py

```diff
@@ -12,35 +13,37 @@
     PLATFORM_SCHEMA as PARENT_PLATFORM_SCHEMA,
     DeviceScanner,
 )
-from homeassistant.const import CONF_HOST
+from homeassistant.const import CONF_HOST, CONF_PASSWORD
 from homeassistant.core import HomeAssistant
 import homeassistant.helpers.config_validation as cv
 from homeassistant.helpers.typing import ConfigType
 
 _LOGGER = logging.getLogger(__name__)
 
-DEFAULT_IP = "192.168.1.1"
+DEFAULT_IP = "internetbox.swisscom.ch"
 
 PLATFORM_SCHEMA = PARENT_PLATFORM_SCHEMA.extend(
-    {vol.Optional(CONF_HOST, default=DEFAULT_IP): cv.string}
+    {
+        vol.Required(CONF_PASSWORD): cv.string,
+        vol.Optional(CONF_HOST, default=DEFAULT_IP): cv.string
+    }
```

Comment:
Schema changes for YAML configuration is no longer allowed. This integration needs to implement a config flow.

See https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md